bad-code.py Test task for Python Mentor второе задание.py

```diff
@@ -0,0 +1,74 @@
+import re
+import datetime
+from collections import OrderedDict
+
+def create_racer_abbreviations_dict(file_name):
+    """Retrieves {'abbreviation': (name, team)}" format dict from abbreviations.txt"""
+    abbreviations_dict = {}
+    with open(file_name, 'r') as fn:
+        for line in fn:
+            matchObj = re.match(r'^(\w+)_([a-zA-Z\s]+)_([a-zA-Z\s]+)$', line)
+            # group(1) is abbreviation, i.e 'SVM'
+            abbreviations_dict[matchObj.group(1)] = (
+                matchObj.group(2),  # name of a pilot
+                matchObj.group(3).rstrip(),  # team
+            )
+    return abbreviations_dict
+
+
+# {'abbreviation of pilot': ('name of pilot, 'team')}
+abbr_dict = create_racer_abbreviations_dict(
+    'abbreviations.txt')
+
+
+# returns timing log from start.log or end.log in {'abbreviation': time} format
+def retrieve_timings_from_log(file_name):
```

Comment:
You need to add:

- annotations to the function
- docstrings for the function parameters and the function's return value. If you're feeling lazy to write docstrings, you can ask ChatGPT to do it for you. Before integrating them into your code, make sure to rewrite them.